go-stash/resources.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2020 The Flux CD contributors.
```

Comment:
Please change this to `Copyright 2021 The Flux authors` note there is no dot at the end.